packages/truffle-box/box.ts

```diff
@@ -1,12 +1,12 @@
-const utils = require("./lib/utils");
-const tmp = require("tmp");
-const path = require("path");
-const Config = require("truffle-config");
-const ora = require("ora");
-const fse = require("fs-extra");
-const inquirer = require("inquirer");
+import utils from "./lib/utils";
+import tmp from "tmp";
+import path from "path";
+import Config from "truffle-config";
```

Comment:
Huh, we don't actually use the typings here, do we?

Reply:
We don't, but I think `tsc` does? Was yelling at me before I added the types.

Reply:
Ah well, cool I guess!